docs/source/_templates/custom-class-template.rst

```diff
@@ -0,0 +1,30 @@
+{{ fullname | escape | underline}}
```

Comment:
Template enables generating documentation for not only classes but also their methods and attributes